modules/images/regenerate-existing-images/admin-page.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Admin page for background jobs
+ *
+ * @since n.e.x.t
+ * @package performance-lab
+ */
+
+/**
+ * Registers a management page for background jobs.
+ *
+ * @since n.e.x.t
+ */
+function perflab_register_background_jobs_page() {
+	add_management_page(
+		__( 'Background Jobs', 'performance-lab' ),
+		__( 'Background Jobs', 'performance-lab' ),
+		'manage_categories',
+		'background-jobs',
+		'perflab_render_background_jobs_page'
+	);
+}
+add_action( 'admin_menu', 'perflab_register_background_jobs_page' );
```

Comment:
I wonder if instead of a custom page we could rather use the regular `edit-tags.php?taxonomy=background_job` screen, even if we added the link to it in a custom place in the admin menu.